Tools/TestRules.ruleset

```diff
@@ -4,14 +4,41 @@
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```

Comment:
Not entirely - this is a combination of exclusions for generated code and for tests.